cmd/snap-confine/seccomp-support.c

```diff
@@ -109,12 +133,82 @@ static void validate_bpfpath_is_safe(const char *path)
 	}
 }
 
+static void sc_cleanup_sock_fprog(struct sock_fprog *prog)
+{
+	free(prog->filter);
+	prog->filter = NULL;
+}
+
+static void sc_must_read_filter_from_file(FILE *file, uint32_t len_bytes, char *what, struct sock_fprog *prog)
+{
+    if (len_bytes == 0) {
+        die("%s filter may only be empty in unrestricted profiles", what);
+    }
```

Comment:
something is off with formatting here, can you run `make fmt` ?

Reply:
I've pushed one more patch applying formatting. We have a bit of trouble with enforcing this properly on top of the codebase using 2 different formatters due to historical reasons. Unfortunately indent 2.2.13 is only available in Ubuntu since 23.10.